entrypoint.sh

```diff
@@ -2,9 +2,20 @@
 
 set -e -o pipefail
 
-if [[ "$(kubectl get clusterrolebinding system:node -o="go-template={{.subjects}}")" == "<no value>" ]]; then
-  echo "Deleting clusterrolebinding/system:node. This is a workaround for https://github.com/kubernetes/kubernetes/pull/60741."
-  kubectl delete clusterrolebinding system:node
+KUBE_RELEASE_VERSION_REGEX="^v(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)(-([a-zA-Z0-9]+)\\.(0|[1-9][0-9]*))?$"
```

Comment:
IMO, it seems no differences between using regexp and awk for getting readability of code.
So anyway ok, I'm not particular about it.

Reply:
I made a mistake the below condition. If the server version is "v1.11.0", it will return "true" :weary:.

```
if [[ "$KUBE_VERSION" < "$KUBE_VERSION_1_11" ]]; then
```

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.0", GitCommit:"91e7b4fd31fcd3d5f436da26c980becec37ceefe", GitTreeState:"clean", BuildDate:"2018-06-27T22:29:25Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.0", GitCommit:"91e7b4fd31fcd3d5f436da26c980becec37ceefe", GitTreeState:"clean", BuildDate:"2018-06-27T20:08:34Z", GoVersion:"go1.10.2", Compiler:"gc", Platform:"linux/amd64"}
$ docker run -e DEBUG=1 -v $PWD:/.kube -e KUBECONFIG=/.kube/config zlabjp/update-storage-objects
Error from server (NotFound): clusterrolebindings.rbac.authorization.k8s.io "system:node" not found
...
```

So please fix it as follows :bow::

```
if [[ $KUBE_VERSION -lt $KUBE_VERSION_1_11 ]]; then
```

Reply:
Done :octocat: